src/software/thunderscope/gl/graphics/gl_circle.py

```diff
@@ -15,16 +15,18 @@ def __init__(
         radius: float = 1,
         num_points: int = 24,
```

Comment:
`num_points` should be passed to the `self.set_radius` call

Reply:
done